druid/src/app.rs

```diff
@@ -38,11 +36,22 @@ pub struct AppLauncher<T> {
     ext_event_host: ExtEventHost,
 }
 
+/// Defines how a windows size should be determined
+#[derive(Copy, Clone, Debug)]
+pub enum WindowSizePolicy {
+    /// Use the content of the window to determine the size
+    Content,
+    /// Use the provided window size
+    User,
```

Comment:
Another option here would be to have this enum variant take a `Size`?

Reply:
I tried that and it got really tangled up with the events from resize.

Reply:
I think that the whole sizing options on WindowConfig need some thought (eg what gets pushed down to the platform, is that the same everywhere?) but I don't want to get bogged down in it here. Its definitely not in its final form, perfect is the enemy of the good.